src/core/public/chrome/chrome_service.tsx

```diff
@@ -391,6 +406,16 @@ export interface ChromeStart {
    */
   setBreadcrumbs(newBreadcrumbs: ChromeBreadcrumb[]): void;
 
+  /**
+   * Get an observable of the current custom nav link
+   */
+  getCustomLink$(): Observable<ChromeCustomLink | undefined>;
+
+  /**
+   * Override the current set of custom nav link
+   */
+  setCustomLink(newCustomLink?: ChromeCustomLink): void;
```

Comment:
Why not more extendable version:`Chrome CustomLink[]`?

Reply:
After discussing with @ryankeairns, we didn't want to allow for multiple links to be registered by plugins and potentially flood the left nav. This is also a temporary addition to the collapsible nav to provide cloud users a link that redirects them to the cloud console and should really only be used by the cloud plugin until the redesign work in #65787 relocates this link.